docs/bridge-guide.md

```diff
@@ -1,56 +1,83 @@
-# Bridge with 🤗Hugging Face
+# Get Started with 🤗 Hugging Face Conversion
 
-Megatron Bridge provides seamless bidirectional conversion between 🤗Hugging Face Transformers and Megatron Core model formats. This guide covers the main APIs for loading models, checking compatibility, and converting between formats.
+Megatron Bridge provides seamless bidirectional conversion between 🤗 Hugging Face Transformers and Megatron-Core model formats. This guide covers the main APIs for loading models, checking compatibility, and converting between formats.
 
-## Loading a 🤗Hugging Face Model into Megatron Implementation
+## Design and Goals
 
-The easiest way to load a 🤗Hugging Face model is using `AutoBridge.from_hf_pretrained()`, which automatically detects the model architecture and selects the appropriate bridge. You can then use `AutoBridge.to_megatron_model()` to initialize the Megatron model from the 🤗Hugging Face configuration and load 🤗HuggingFace weights.
+- Single high-level entry point: `AutoBridge` detects HF model architectures and dispatches to the correct bridge.
+- Bidirectional conversion: Import HF → Megatron for training; export Megatron → HF for deployment.
+- Parallelism-aware: Handles TP/PP/VPP/CP/EP/ETP distributions during conversion.
+- Streaming and memory efficiency: per-parameter streaming using safetensors.
+- Provider pattern: Configure Megatron-Core `TransformerConfig`-compatible attributes before instantiation via `to_megatron_provider()`.
+- Convenience workflows: `import_ckpt` and `export_ckpt` provide one-call HF↔Megatron checkpoint flows.
+
+See the repository `README.md` for installation, supported models, and project highlights.
+
+## Loading a 🤗 Hugging Face Model into Megatron
+
+The easiest way to load a 🤗 Hugging Face model is using `AutoBridge.from_hf_pretrained()`, which automatically detects the model architecture and selects the appropriate bridge for conversion. You can then use `AutoBridge.to_megatron_model()` to initialize the Megatron model from the 🤗 Hugging Face configuration and load 🤗 Hugging Face weights at the same time.
 
 ### Basic Usage
 
+```mermaid
+flowchart LR
+    HF[Hugging Face<br/>PreTrained Model/Config] -->|detect| AutoBridge
+    AutoBridge -->|select| Bridge[MegatronModelBridge<br/>architecture-specific]
+    Bridge -->|provider bridge| Provider[Model Provider<br/>TransformerConfig builder]
+    Provider -->|instantiate| Megatron[Distributed Megatron Model]
+    Bridge -->|mapping registry| Registry[MegatronMappingRegistry]
+    Registry -->|maps to| Mappings[Param Mapping<br/>Auto Row Col QKV ...]
+    HF <-->|convert per-parameter| Bridge
+    Megatron --|TP/PP/VPP-aware|--> Mappings
+```
```

Comment:
@yaoyu-33 this was added for testing that the mermaid diagram rendering worked. it shouldn't be added here